process/track/baseBlockTrack_test.go

```diff
@@ -2495,3 +2557,34 @@ func TestBaseBlockTrack_DoWhitelistWithShardHeaderIfNeededMetaShouldWhitelistCro
 
 	assert.True(t, ok)
 }
+
+func TestBaseBlockTrack_IsHeaderOutOfRangeShouldWork(t *testing.T) {
```

Comment:
The tests run under the assumption that `lastCrossNotarizedHeader` has round `0`, nonce `0`. Is this correct?

Reply:
Yes, what I wanted to test is that if the difference between last cross notarized header, no matter if it has nonce 0 (genesis),  and the current header received is out of range ( <= last notarized or  > last notarized + window ) the method works correctly.

Reply:
Anyway, I have refactored the test to be more clear now